lib/middlewares/translate_errors.ex

```diff
@@ -1,31 +1,34 @@
-defmodule Crudry.Middlewares.HandleChangesetErrors do
+defmodule Crudry.Middlewares.TranslateErrors do
   @moduledoc """
-  Absinthe Middleware to translate changeset errors into human readable messages. It supports nested errors.
+  Absinthe Middleware to translate errors and changeset errors into human readable messages using [Gettext](https://github.com/elixir-lang/gettext).
 
   ## Usage
 
-  To handle errors for a field, add it after the resolve, using [`middleware/2`](https://hexdocs.pm/absinthe/Absinthe.Middleware.html#module-the-middleware-2-macro):
+  `Cudry.Translator` is used by default to translate error messages to the default locale `en`. You can also use your own Gettext module by adding it and the desired `locale` to to your Absinthe's schema `context/1` function:
 
-      alias Crudry.Middlewares.HandleChangesetErrors
+    def context(context) do
+      context
+      |> Map.put(:translator, MyApp.Translator)
+      |> Map.put(:locale, "pt_BR")
```

Comment:
Wouldn't the default locale already be defined in `MyApp.Translator`?

Reply:
Yes, indeed. I just wanted to show that the default locale can be overridden.

Should I remove this line from the docs?

Reply:
Maybe separate this example into 2?